network/f5/bigip_virtual_server.py

```diff
@@ -508,6 +509,15 @@ def set_default_persistence_profiles(api, name, persistence_profile):
                 profiles=[[{'profile_name': persistence_profile, 'default_profile': True}]]
             )
             updated = True
+        # If a virtual server has a persistence profile and would like this to be None this section enabled this
+        # When persistence_profile is not defined in task or set to Null in vars the persistence profile will be
+        # set to None.
+        if persistence_profile is None or False:
```

Comment:
I've always subscribed to the notion of the playbook containing the state you want to implement, rather than using verb-type attributes to trigger things like 'go delete this thing'. Just explicitly set that thing _to_ empty. If you don't want to inspect or implement state on an attribute, don't include it in the playbook.